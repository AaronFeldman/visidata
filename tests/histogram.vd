sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
		numeric_binning	set-option	True		
benchmark	Quantity		type-int		#	
benchmark	Quantity		addcol-expr	Quantity > 1	=	
benchmark	Quantity > 1		select-col-regex	True	|	
benchmark			dup-selected		"	
benchmark_selectedref	Quantity		freq-col		F	
