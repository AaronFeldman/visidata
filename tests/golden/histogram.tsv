Quantity	count	percent	histogram
4	10	35.71	*****
2	4	14.29	**
30	3	10.71	*
5	3	10.71	*
3	3	10.71	*
6	2	7.14	*
12	1	3.57	
50	1	3.57	
144	1	3.57	
